.gitlab-ci.yml

```diff
@@ -342,25 +342,16 @@ init_workspace:
     - sed -i 's/false/true/g' /etc/default/sysstat
     - service sysstat start
     - sar -P ALL 2 -o /var/log/sysstat/sysstat.log -uqrbS >/dev/null 2>&1 &
-    # Install gcsfuse and mount shared Bitbake sstate-cache
-    - echo "deb http://packages.cloud.google.com/apt gcsfuse-`lsb_release -c -s` main" | tee /etc/apt/sources.list.d/gcsfuse.list
-    - curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-    - apt-get update
-    - apt-get install -qqy gcsfuse
-    - groupadd -f fuse
-    - usermod -a -G fuse mender
-    - mkdir -p /mnt/sstate-cache
-    - chown mender:mender /mnt/sstate-cache
-    - echo "user_allow_other" >> /etc/fuse.conf
-    # Mout GCS bucket as sstate-cache
-    - gcsfuse --uid 1010 --gid 1010 --limit-ops-per-sec 10000 -o allow_other sstate-cache /mnt/sstate-cache
     # Setup KVM
     - usermod -a -G kvm mender
     - apt-get install -qqy kmod libvirt-bin qemu-utils qemu-kvm
     - apt-get install -qqy linux-modules-`uname -r`
     # Enable nesting VMs
     - modprobe -r kvm_intel
     - modprobe kvm_intel nested=Y
+    - apt-get install nfs-common
+    - mkdir -p /mnt/sstate-cache
+    - mount.nfs4 10.162.0.2:/export/sstate-cache /mnt/sstate-cache
```

Comment:
Where is the IP address coming from? Should it be a variable?

Reply:
I've allocated a fixed internal (and external) ip address for the nfs server (only IP ranges corresponding to internal addresses are granted read-write permissions to the cache). I'll make it a variable definition.